components/results_table/commons/results_table_base.lua

```diff
@@ -156,6 +163,7 @@ function BaseResultsTable:queryData()
 	})
 
 	if type(data) ~= 'table' then
+		mw.logObject(self:buildConditions(), 'conditions')
```

Comment:
Intended?

Reply:
yes
it is basically to show the conditions if it should error

Reply:
Fair, github didn't show me the error below.